packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -269,19 +292,37 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                                     className='p2p-my-ads__table-popovers__activate'
                                     message={localize('Activate')}
                                 >
-                                    <Icon icon='IcUnarchive' color={general_store.is_barred && 'disabled'} size={16} />
+                                    <Icon icon='IcUnarchive' color={general_store.is_barred && 'disabled'} />
```

Comment:
What would the default color be if general_store.is_barred is falsy?

Reply:
The icons have a default color of `--fill-color1`